server/routes/api/conversations.js

```diff
@@ -79,4 +80,28 @@ router.get("/", async (req, res, next) => {
   }
 });
 
+// Accepts an array of conversations to update the value of
+router.patch("/read", async (req, res, next) => {
+  try {
+    if (!req.user) {
+      return res.sendStatus(401);
+    }
+
+    const userId = req.user.id;
+    const { conversationIds } = req.body;
+
+    if (conversationIds) {
```

Comment:
Because `conversationIds` is an array, this is always going to be truthy, even if it's empty. I would suggest using the length to check:

```suggestion
    if (conversationIds.length) {
```